src/modules/alwaysontop/AlwaysOnTop/AlwaysOnTop.vcxproj.filters

```diff
@@ -115,13 +115,14 @@
     <ClInclude Include="ScalingUtils.h">
       <Filter>Header Files</Filter>
     </ClInclude>
-	<ClInclude Include="Generated Files/resource.h">
+    <ClInclude Include="Generated Files/resource.h">
       <Filter>Generated Files</Filter>
     </ClInclude>
+    <ClInclude Include="NotificationUtil.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
   </ItemGroup>
   <ItemGroup>
-    <ResourceCompile Include="AlwaysOnTop.rc">
```

Comment:
Surprisingly, this file is an unused duplicate. AlwaysOnTop.base.rc is the actual file that gets used.